panopticum/templates/widget/component_owners.html

```diff
@@ -14,12 +15,15 @@ <h2>Owners</h2>
               <tr>
                 <td class='owner'>
                   <div class='block'>
-                    <div class='image'><img v-bind:src="component.latest_version.owner_maintainer.photo"></div>
+                    <div class='image'>
+                      <img v-bind:src="component.latest_version.owner_maintainer.photo" v-if="component.latest_version.owner_maintainer">
+                      <img v-else src="{% static '/images/default_avatar.png' %}">
```

Comment:
I remember we agreed not to use Django templating directives inside w/o a need to simplify the switch to SPA later. No?

Reply:
Yep here is some inconsistency. But i prefer regard that like ```{% extend ... %}``` django directives. We can agree that is not templating directive. Lets regard that as service directive. We will replace it to something ```<img :src="`${publicPath}my-image.png`">``` anyway at SPA stage.
https://cli.vuejs.org/guide/html-and-static-assets.html#the-public-folder